local-cli/runAndroid/runAndroid.js

```diff
@@ -156,6 +156,20 @@ function buildAndRun(args) {
       'utf8'
     ).match(/package="(.+?)"/)[1];
 
+    const variantName = args.variant || 'debug';
```

Comment:
The config that specifies options for a command has a field `default`. Use it to specify `debug` value.